File: app/src/main/java/com/snj/letschat/utils/SharedPrefConfigUtils.java
package com.snj.letschat.utils;

import android.content.Context;
import android.content.SharedPreferences;

public class SharedPrefConfigUtils {
    public static final String MY_PREFERENCES = "MyPrefs";
    public static final String USER_ID = "uIdKey";
    public static final String USER_NAME = "uNameKey";
    public static final String USER_EMAIL = "uEmailKey";
    public static final String USER_IMAGE = "uImageKey";

    public static final long getUserId(Context context){
        return context.getSharedPreferences(SharedPrefConfigUtils.MY_PREFERENCES,Context.MODE_PRIVATE).getLong(SharedPrefConfigUtils.USER_ID,1);
    }
    public static String  getUserEmailId(Context context){
        return context.getSharedPreferences(SharedPrefConfigUtils.MY_PREFERENCES,Context.MODE_PRIVATE).getString(SharedPrefConfigUtils.USER_EMAIL,"");
    }
    public static String  getUserName(Context context){
        return context.getSharedPreferences(SharedPrefConfigUtils.MY_PREFERENCES,Context.MODE_PRIVATE).getString(SharedPrefConfigUtils.USER_NAME,"");
    }


    public static SharedPreferences getSharedPreference(Context context) {
        return context.getSharedPreferences(SharedPrefConfigUtils.MY_PREFERENCES, Context.MODE_PRIVATE);
    }
}

File: app/src/main/java/com/snj/letschat/utils/SharedPrefConfigUtils.kt
package com.snj.letschat.utils

import android.content.Context
import android.content.SharedPreferences

class SharedPrefConfigUtils {
    companion object {
        val MY_PREFERENCES = "MyPrefs"
        val USER_ID = "uIdKey"
        val USER_NAME = "uNameKey"
        val USER_EMAIL = "uEmailKey"
        val USER_IMAGE = "uImageKey"

        fun getUserId(context: Context): Long {
            return context.getSharedPreferences(MY_PREFERENCES, Context.MODE_PRIVATE).getLong(USER_ID, 1)
        }

        fun getUserEmailId(context: Context): String {
            return context.getSharedPreferences(MY_PREFERENCES, Context.MODE_PRIVATE).getString(USER_EMAIL, "")
        }

        fun getUserName(context: Context): String {
            return context.getSharedPreferences(MY_PREFERENCES, Context.MODE_PRIVATE).getString(USER_NAME, "")
        }

        fun getSharedPreference(context: Context): SharedPreferences {
            return context.getSharedPreferences(MY_PREFERENCES, Context.MODE_PRIVATE)
        }
        fun clear(context: Context) {
            var editor = getSharedPreference(context).edit()
            editor.clear()
            editor.commit()
        }
    }
}